VideoLocker/assets/config/profiles.json

```diff
@@ -0,0 +1,56 @@
+{
+    "fields":[
+    {
+              "name" : "account_privacy",
+              "label" : "edX learners can see my:",
+              "type" : "switch",
+              "options" : {
+                "values" : [
+                            { "name" : "Full Profile", "value" : "all_users" },
+                            { "name" : "Limited Profile",  "value" : "private" }
+                 ]
+              },
+              "instructions" : "A limited profile only shares your username, though you can still choose to add your own profile photo.",
```

Comment:
@bguertin I think the limited profile shares both username and profile photo (as confirmed by @nasthagiri in Hipchat yesterday). 
"A limited profile shares your username and a profile image if you have added one." @explorerleslie ?